digdag-standards/src/main/java/io/digdag/standards/operator/MailOperatorFactory.java

```diff
@@ -229,4 +256,44 @@ private InternetAddress newAddress(String str)
             }
         }
     }
+
+    private static Optional<SmtpConfig> systemSmtpConfig(Config systemConfig)
+    {
+        Optional<String> host = systemConfig.getOptional("config.mail.host", String.class);
+        if (!host.isPresent()) {
+            return Optional.absent();
+        }
+        SmtpConfig config = ImmutableSmtpConfig.builder()
+                .host(host.get())
+                .port(systemConfig.get("config.mail.port", int.class))
+                .startTls(systemConfig.get("config.mail.tls", boolean.class, true))
```

Comment:
Of course I know this name isn't changed in this pull request, but it seems better to be `starttls` not just `tls`...